UIKitDZ/UIKitDZ/Notifications/NotificationsViewController.swift

```diff
@@ -0,0 +1,284 @@
+        view.addSubview(requestLabel)
+        requestLabel.text = AppConstants.requestTitle
+        requestLabel.font = UIFont.systemFont(ofSize: 14)
+        requestLabel.translatesAutoresizingMaskIntoConstraints = false
+        requestLabel.widthAnchor.constraint(equalToConstant: 251).isActive = true
+        requestLabel.heightAnchor.constraint(equalToConstant: 17).isActive = true
+        requestLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 17).isActive = true
+        requestLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: 150).isActive = true
+    }
+
+    private func configureTable() {
+        view.addSubview(tableView)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.estimatedRowHeight = 144
+        tableView.separatorStyle = .none
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
+        tableView.topAnchor.constraint(equalTo: requestLabel.bottomAnchor, constant: 5).isActive = true
+        tableView.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
+        tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+    }
+
+    private func configTodayHeader() {
+        todayHeaderLabel.text = AppConstants.todayHeader
+        todayHeaderLabel.font = UIFont.systemFont(ofSize: 14, weight: .bold)
+        todayHeaderLabel.textColor = .black
+    }
+
+    private func configWeekHeader() {
+        weekHeaderLabel.text = AppConstants.thisWeekHeader
+        weekHeaderLabel.font = UIFont.systemFont(ofSize: 14, weight: .bold)
+        weekHeaderLabel.textColor = .black
+    }
+
+    private func registerCell() {
+        tableView.register(
+            NotificationCommentTableViewCell.self,
+            forCellReuseIdentifier: AppConstants.commentIdentifier
+        )
+        tableView.register(
+            NotificationSubscriptionTableViewCell.self,
+            forCellReuseIdentifier: AppConstants.SubscriptionIdentifier
+        )
+    }
+}
+
+// MARK: - Extension UITableViewDataSource
+
+extension NotificationsViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        2
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        let todayNotification = cellModels.filter {
+            guard
+                let actiontime = $0.notificationTime,
+                actiontime == " 12 ч"
+            else { return false }
+
+            return true
+        }
+
+        let anohterNotification = cellModels.filter {
+            guard
+                let actiontime = $0.notificationTime,
+                actiontime != " 12 ч"
+            else { return false }
+
+            return true
+        }
+
+        switch section {
+        case 0:
```

Comment:
```suggestion
        case 0:
```
хардкод это плохо поробуй на 2 рейса замени место 0 и 1